homeassistant/components/deako/const.py

```diff
@@ -0,0 +1,15 @@
+"""Constants for Deako."""
+# Base component constants
+NAME = "Deako"
+DOMAIN = "deako"
+DOMAIN_DATA = f"{DOMAIN}_data"
+
+# Icons
+ICON = "mdi:format-quote-close"
```

Comment:
This is only used in light.py now. `const.py` is used for constants used in more files so please move this to `light.py`.

Reply:
Got it, thanks!